lib/NativeCall.rakumod

```diff
@@ -260,6 +263,144 @@ sub resolve-libname($libname) {
     $libname.platform-library-name.Str
 }
 
+use MONKEY-SEE-NO-EVAL;
+my $use-dispatcher = so $*RAKU.compiler.?supports-op('dispatch_v') && EVAL q:to/CODE/;
+    use nqp;
+    sub raku-native-dispatch-deproxy($callee, *@args) {
+        $callee(|@args) # implicitly fetches values from proxies
+    }
```

Comment:
Uh, that's clever! Not so runtime efficient as the proxy reader approach, but... :-) As a nitpick, I wonder if `**` would be better (so we don't flatten anything away)?